dts/bindings/memory-controllers/renesas,smartbond-nor-psram.yaml

```diff
@@ -0,0 +1,254 @@
+# Copyright (c) 2023 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Renesas Smartbond(tm) NOR/PSRAM controller
+
+include: base.yaml
+
+compatible: "renesas,smartbond-nor-psram"
+
+properties:
+  reg:
+    required: true
+
+  is-ram:
+    type: boolean
+    description: |
+      If present, the memory controller will be configured to drive PSRAM devices.
+
+  dev-size:
+    type: int
+    required: true
+    description: |
+      Memory size/capacity in bits.
+
+  dev-type:
+    type: int
+    default: 0x5D
+    description: |
+      Device type, part of device ID, used for verification during MEMC initialization.
```

Comment:
all default values in bindings require justification (https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values)

ditto for all the other properties with default values in this file